lib/scarpe/wv/shape_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ShapeHelper
```

Comment:
i wanted move_to , line_to commands in nrml file and shape_path commmand in wv directory so i made a seperate module for these methods

Reply:
Sounds good. We'll probably want more of that kind of thing going forward. I've been looking at shoes-core in shoes4 and it does quite a lot of it...